devtools/SDS - Local Docker.postman_collection.json

```diff
@@ -82,7 +82,7 @@
         "header": [],
         "body": {
           "mode": "raw",
-          "raw": "{\n  \"survey_id\": \"NRX\",\n  \"period_id\": \"ttt\",\n  \"form_type\": \"yyy\",\n  \"title\": \"Which side was better?\",\n  \"sds_schema_version\": 4,\n  \"schema_version\": \"v1.0.0\",\n  \"data\": [\n    {\n      \"ruref\": \"43532\",\n      \"runame\": \"Pipes and Maps Ltd\",\n      \"local_unit\": [\n        {\n          \"luref\": \"2012763A\",\n          \"luname\": \"Maps Factory\"\n        },\n        {\n          \"luref\": \"20127364B\",\n          \"luname\": \"Pipes R Us Subsidiary\"\n        }\n      ]\n    },\n    {\n      \"ruref\": \"65871\",\n      \"runame\": \"Boats and Floats Ltd\",\n      \"local_unit\": [\n        {\n          \"luref\": \"2012763A\",\n          \"luname\": \"Arms Factory\"\n        },\n        {\n          \"luref\": \"20127364B\",\n          \"luname\": \"Swords Subsidiary\"\n        },\n        {\n          \"luref\": \"20127365C\",\n          \"luname\": \"Armor N Things\"\n        }\n      ]\n    }\n  ]\n}",
```

Comment:
I think \"form_type\": \"yyy\",\n  should all be removed

Reply:
removed :)